From 1148e4ba78d2bdb36bde17fd0991d99a6794a9c6 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Tue, 6 Dec 2016 12:25:52 -0500
Subject: [PATCH 0221/2094] drm/amd/display: Fix warning. Set MAX_SURFACES to 3

Currently we never use more than 2 surfaces in all use cases.
This avoids pushing dc_commit_surfaces_to_target beyond a
stacksize of 1024.

Change-Id: I3db9562a99057810c5fda5c388ce83f6e5a899ed
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Tony Cheng <Tony.Cheng@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dc.h b/drivers/gpu/drm/amd/display/dc/dc.h
index 6bb3aeb..5f60800 100644
--- a/drivers/gpu/drm/amd/display/dc/dc.h
+++ b/drivers/gpu/drm/amd/display/dc/dc.h
@@ -34,7 +34,7 @@
 #include "link_service_types.h"
 
 #define MAX_TARGETS 6
-#define MAX_SURFACES 6
+#define MAX_SURFACES 3
 #define MAX_SINKS_PER_LINK 4
 
 /*******************************************************************************
-- 
2.7.4

